react.js

```diff
@@ -11,13 +22,20 @@ stuffArray.map(function(el, i){
   console.log(el + " is at index: " + i)
 })
 
+
 // your for loop here, you can use the same stuffArray array
+var stuffArray = [42, "Arthur Dent", 1978, "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
+for(let i = 0; i < stuffArray.length; i++)
 
```

Comment:
The loop looks good, but go ahead and build out the function that returns... el + " is at index: " + i